androidApp/src/main/java/com/mbta/tid/mbta_app/android/state/subscribeToAlerts.kt

```diff
@@ -61,7 +63,7 @@ fun subscribeToAlerts(
     val viewModel: AlertsViewModel = viewModel(factory = AlertsViewModel.Factory(alertsRepository))
 
     LifecycleResumeEffect(key1 = null) {
-        viewModel.connect()
+        CoroutineScope(Dispatchers.IO).launch { viewModel.connect() }
```

Comment:
@JackVCurtis added the use of Dipsatchers.IO in https://github.com/mbta/mobile_app/pull/557, and I mistakenly removed in https://github.com/mbta/mobile_app/pull/588. 

This addresses some of the jankiness / lag while panning the map or scrolling the list of departures.